modules/aptx/aptx.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file aptx.c aptX Audio Codec
+ *
+ * Copyright (C) 2019 Hessischer Rundfunk
+ */
+
+#include <re.h>
+#include <baresip.h>
+#include "aptx.h"
+
+/**
+ * @defgroup aptx aptx
+ *
+ * aptX audio codec (Standard and HD variant)
+ *
+ * Supported version:
+ *      libopenaptx 0.1.0 or later
+ *
+ * References:
+ *
+ *    RFC 7310  RTP Payload Format for Standard apt-X and Enhanced apt-X Codecs
+ *
+ * 
+ * TODOs:
+ *    - Code check, cleanup and error handling
+ *    - Add SDP fmt negotiation & config preconfiguration
+ *    - Check and implement other sampling rates and channel modes
+ *    - Add real 24 bit I/O support
+ *    
+ */
+
+static struct aucodec aptx = {
+	.name = "aptx",
+	.srate = APTX_SRATE,
+	.crate = APTX_SRATE,
+	.ch = APTX_CHANNELS,
+	.pch = APTX_CHANNELS,
+	//.ptime = 4,
+	.encupdh = aptx_encode_update,
+	.ench = aptx_encode_frm,
+	.decupdh = aptx_decode_update,
+	.dech = aptx_decode_frm,
+	.fmtp_ench = aptx_fmtp_enc,
+	.fmtp_cmph = aptx_fmtp_cmp,
+};
+
+static int module_init(void)
+{
+	aucodec_register(baresip_aucodecl(), &aptx);
+
+	return 0;
+}
+
```

Comment:
please use 2 empty lines between functions.

Reply:
OK, thanks for your input!
This was Visual Studio Code autoformat.
Is there any standard named codestyle preset or plugin I should set the autoformater to?

Anyway I will reformat it as required of course.